utility/request-parser.js

```diff
@@ -0,0 +1,26 @@
+const _ = require('lodash');
+const queryString = require('query-string');
+
+const validateObjectTextProperty = (validationForProperty, textObjectProperty) => {
+  return _.every(_.values(validationForProperty), validation => validation(textObjectProperty));
+}
+
+const isObjectTextValid = (textObject, objectWithValidations) => {
+  return _.every(_.map(_.keys(objectWithValidations), (key) => validateObjectTextProperty(objectWithValidations[key],textObject[key])));
```

Comment:
You don't need to do `_.map(_.keys())`. You get multiple loops with such structure. Use reduce instead

```
_.every(_.reduce(textObject, (validatedProperties, userInputProperty, propertyKey) => (
  validateObjectTextProperty(userInputProperty, objectWithValidators[propertyKey])
), [])
```

Or you can even go simpler and do just

```
_.reduce(textObject, (isObjectValid, userInputProperty, propertyKey) => (
  validateObjectTextProperty(userInputProperty, objectWithValidators[propertyKey]) && isObjectValid
), true)
```

Reply:
I would also try to simplify the naming of variable. Variables are a little bit confusing.